x-pack/plugins/lens/public/datasources/form_based/operations/definitions/filters/filter_popover.test.tsx

```diff
@@ -14,16 +14,21 @@ import { FilterPopover } from './filter_popover';
 import { LabelInput } from '../shared_components';
 import { QueryStringInput } from '@kbn/unified-search-plugin/public';
 import { QueryInput } from '@kbn/visualization-ui-components';
+import { Query } from '@kbn/es-query';
 
-jest.mock('.', () => ({
-  isQueryValid: () => true,
-  defaultLabel: 'label',
-}));
+jest.mock('.', () => ({}));
```

Comment:
Is this needed?

Reply:
Yes, we have some circular dependencies here.